scripts/portstat

```diff
@@ -204,6 +205,76 @@ def parse_bcmcmd_ps(bcmcmd_output):
     bcmcmd_lines = bcmcmd_output.split("\n")
 
 
+def get_cnstat(cnstat_dir):
+    """
+          Get the cnstat from netstat.
+    """
+    try:
+        bcmcmd_output = subprocess.Popen(([BCMCMD_PATH,
+                                           'show counters changed same nz']),
+                                         stdout=subprocess.PIPE,
+                                         shell=False).communicate()[0]
+    except EnvironmentError as e:
+        print e, e.errno
+        sys.exit(e.errno)
+
+    bcm_dict = parse_bcmcmd_output(bcmcmd_output)
+
+    try:
+        netstat_out = subprocess.Popen(([NETSTAT_PATH, '-i']),
+                                       stdout=subprocess.PIPE,
+                                       shell=False).communicate()[0]
+    except EnvironmentError as e:
+        print e, e.errno
+        sys.exit(e.errno)
+
+    netstat_lines = netstat_out.split("\n")
+
+    # Since netstat -i returns some stats as 32-bits, get full 64-bit
+    # stats from /prov/net/dev and display only the 64-bit stats.
+    try:
+        proc_out = subprocess.Popen(([CAT_PATH, '/proc/net/dev']),
+                                    stdout=subprocess.PIPE,
+                                    shell=False).communicate()[0]
+    except EnvironmentError as e:
+        print e, e.errno
+        sys.exit(e.errno)
+
+    proc = {}
+    for line in proc_out.split("\n"):
+        parsed = re.findall("\s*([^ ]+):(.*)", line)
+        if not parsed:
+            continue
+        iface, stats = parsed[0]
+        proc[iface] = stats.split()
+
+    # At this point, either we'll create a file or open an existing one.
+    if not os.path.exists(cnstat_dir):
+        try:
+            os.makedirs(cnstat_dir)
+        except IOError as e:
+            print e.errno, e
+            sys.exit(1)
```

Comment:
here you only create a folder, but you do not use it anyway in the function. what's the purpose here? should we move it to some where else?